hazelcast/src/main/java/com/hazelcast/instance/DefaultNodeContext.java

```diff
@@ -76,63 +75,58 @@ public AddressPicker createAddressPicker(Node node) {
         }
         ClassLoader classLoader = config.getClassLoader();
         String classname = memberAddressProviderConfig.getClassName();
-        Class<?> clazz = loadMemberAddressProviderClass(classLoader, classname);
+        Class<? extends MemberAddressProvider> clazz = loadMemberAddressProviderClass(classLoader, classname);
+        ILogger memberAddressProviderLogger = node.getLogger(clazz);
```

Comment:
I wish we could end this logger nonsense. Just static configured global logging implementation.